src/engagement_db_to_analysis/traced_data_filters.py

```diff
@@ -65,45 +65,41 @@ def rqa_time_range_filter(user, messages_traced_data, pipeline_config):
     return filtered
 
 
-def filter_test_participants(user, participants_traced_data_map, test_participant_uuids):
+def filter_test_participants_messages(user, messages_traced_data, test_participant_uuids):
     """
     Filters a dict of participants who are not in pipeline_config.test_contacts e.g AVF/Aggregator staff
```

Comment:
`pipeline_config.test_contacts` -> `pipeline_config.test_participant_uuids`

Reply:
I suggest updating to sth like "Filters out test messages"

Reply:
updated thanks.